CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+### Added
+
+- **WIP** Support for `Carthage` dependencies in `Dependencies.swift` [#2394](https://github.com/tuist/tuist/pull/2394) by [@laxmorek](https://github.com/laxmorek).
```

Comment:
Shouldn't it be `SPM` instead of `Carthage`? :D

Reply:
right 😅